doc/source/configuration/index.rst

```diff
@@ -18,3 +18,4 @@ the various features provided.
    wazuh
    vault
    magnum-capi
+   security
```

Comment:
If we have a plain `security` page, we could arguably move wazuh under it. Otherwise, maybe ``cis`` or ``security-hardening`` would be more appropriate?

Reply:
done